e2e/pages/enterChildren/enterChildren.js

```diff
@@ -0,0 +1,139 @@
+		},
+		child1KeyDates: '#children_firstChild_enterChildren_keyDates',
+		child1CareAndContactPlan: '#children_firstChild_enterChildren_careAndContact',
+		child1AdoptionNo: '#children_firstChild_enterChildren_adoption-No',
+		child1MothersName: '#children_firstChild_enterChildren_mothersName',
+		child1FathersName: '#children_firstChild_enterChildren_fathersName',
+		child1FatherResponsibleDropdown: {
+			selector: '#children_firstChild_enterChildren_fathersResponsibility',
+			option: 'Yes'
+		},
+		child1SocialWorkerName: '#children_firstChild_enterChildren_socialWorkerName',
+		child1SocialWorkerTel: '#children_firstChild_enterChildren_socialWorkerTel',
+		child1AdditionalNeedsNo: '#children_firstChild_enterChildren_additionalNeeds-No',
+		child1ContactHiddenNo: '#children_firstChild_enterChildren_detailsHidden-No',
+		child1LitigationNo: '#children_firstChild_enterChildren_litigationIssues-No',
+		addChildButton: 'Add new',
+
+		child2FullName: "#children_additionalChildren_0_enterChildren_childName",
+		child2DOB: {
+			day: '#children_additionalChildren_0_enterChildren_childDOB-day',
+			month: '#children_additionalChildren_0_enterChildren_childDOB-month',
+			year: '#children_additionalChildren_0_enterChildren_childDOB-year'
+		},
+		child2GenderDropdown: {
+			selector:'#children_additionalChildren_0_enterChildren_childGender',
+			option: 'Boy'
+		},
+		child2SituationDropdown: {
+			selector: '#children_additionalChildren_0_enterChildren_livingSituation',
+			option: 'Living with respondents',
+			dateStartedStaying: {
+				day: '#children_additionalChildren_0_enterChildren_situationDate-day',
+				month: '#children_additionalChildren_0_enterChildren_situationDate-month',
+				year: '#children_additionalChildren_0_enterChildren_situationDate-year'
+			},
+			addressOfChild: '#children_additionalChildren_0_enterChildren_situationAddress',
+		},
+		child2KeyDates: '#children_additionalChildren_0_enterChildren_keyDates',
+		child2CareAndContactPlan: '#children_additionalChildren_0_enterChildren_careAndContact',
+		child2AdoptionNo: '#children_additionalChildren_0_enterChildren_adoption-No',
+		child2MothersName: '#children_additionalChildren_0_enterChildren_mothersName',
+		child2FathersName: '#children_additionalChildren_0_enterChildren_fathersName',
+		child2FatherResponsibleDropdown: {
+			selector: '#children_additionalChildren_0_enterChildren_fathersResponsibility',
+			option: 'Yes'
+		},
+		child2SocialWorkerName: '#children_additionalChildren_0_enterChildren_socialWorkerName',
+		child2SocialWorkerTel: '#children_additionalChildren_0_enterChildren_socialWorkerTel',
+		child2AdditionalNeedsNo: '#children_additionalChildren_0_enterChildren_additionalNeeds-No',
+		child2ContactHiddenNo: '#children_additionalChildren_0_enterChildren_detailsHidden-No',
+		child2LitigationNo: '#children_additionalChildren_0_enterChildren_litigationIssues-No',
+	},
+
+	halfFillForm() {
```

Comment:
Page objects should expose API to the application page. That API should in readable manner communicate what actions are available on the page. Having methods like `halfFillForm` and `fillForm` does not communicate intent too well. It also makes it more difficult to reuse.